OSDOCS-16409: Cluster Admin CQAs (#116629)
openshift/openshift-docs: modules/rosa-nodes-machinepools-upgrade-requirements-hcp.adoc

// Module included in the following assemblies:
//
// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

:_mod-docs-content-type: CONCEPT
[id="rosa-nodes-machinepools-upgrade-requirements-hcp_{context}"]
= Machine pool upgrade requirements

[role="_abstract"]

> Each machine pool in a {product-title} cluster upgrades independently. Because the machine pools upgrade independently, they must remain within two minor (Y-stream) versions of the hosted control plane. For example, if your hosted control plane is 4.16.z, your machine pools must be at least 4.14.z.

Reviewer: With an attribute it's already 299 characters long and will go over in a rendered text. Can be tightened by removing the repetition:

```suggestion
Each machine pool in a {product-title} cluster upgrades independently and therefore must remain within two minor (Y-stream) versions of the hosted control plane. For example, if your hosted control plane is 4.16.z, your machine pools must be at least 4.14.z.
```
